### ISSUE TITLE
Tweaked petroleum config required

### ISSUE THREAD

**Issue by Srdjan-V** [juraj-hrivnak/Underdog#34]
By default Tweaked petroleum removes IP reservoir loading, this is intended behavior but was poorly expand.
The cf page has be updated to reflect that and i have created a starting [template](https://github.com/Srdjan-V/TweakedPetroleum/blob/master/examples/default.zs), the configs are equivalent to IP's

**Comment by juraj-hrivnak** [comment]
Thanks for the report, (and making the mod.) I will add the config for the next update.

**Comment by Srdjan-V** [comment]
Than you for using it. I aslo wanted to tell you to make the update quickly, since without the reservoirs being loaded all chunks with fluid chunks will be lost on world load, or tell the players to disable the overwrites in the config (that way the chunks will not be lost, but the will also not be usable)

**Comment by juraj-hrivnak** [comment]
Oh, so only newly generated chunks will have reservoirs or some chunks will get corrupted after the change? 
I hope it’s not the second option.

**Comment by Srdjan-V** [comment]
Not exactly, IP has a way of loading reservoirs through its config. Witch, I have disabled because those configs do not include the additional data I have added (and for consistency). The way I have set it out to be is so that you add the reservoirs through zenscript. But if you do not add the reservoirs all chunks that have a fluid chunk will be lost. Since the game does not know what fluid should be in that chunk. If that hapens the best option is to delete the data for fluid chunks, save/data/immersivepetroleum.data


But as for compatability its fully compatible with the allredy generated reservoirs if you register them with the same name(like in the template i made).
